v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Parallel Statement Execution
+summary: The parallel statement execution feature allows parallel execution of multiple independent SQL statements within a transaction.
+toc: false
+---
+
+CockroachDB supports parallel execution of independent [`INSERT`](insert.html), [`UPDATE`](update.html), [`UPSERT`](upsert.html), and [`DELETE`](delete.html) statements within a single [transaction](https://www.cockroachlabs.com/docs/dev/transactions.html). Executing statements in parallel helps reduce aggregate latency and improve performance. 
+
+<div id="toc"></div>
+
+## Why Use Parallel Statement Execution
+
+To understand why parallel statement execution is required, let's consider a sample scenario to demonstrate how SQL statements are executed. 
+
+Consider a scenario where a user's last name, favorite movie, and favorite song are to be updated on a social networking application. Suppose the database has three tables: username, movies, and songs, that need to be updated to update the user information. Also suppose that the user's ID is 1. Then the traditional transaction to update the user's information is as follows:
+
+~~~ sql
+> BEGIN;
+> UPDATE username SET last_name = 'Smith' WHERE id = 1;
+> UPDATE movies SET favorite_movie = 'The Matrix' WHERE id = 1;
+> UPDATE songs SET favorite_song = 'All this time' WHERE id = 1;
+> COMMIT;
+~~~
+
+In this case, the statements are executed sequentially. Sequential execution of SQL statements returns a return value immediately from the server. After the return value is received, the next SQL statement is executed. The following conceptual diagram shows the sequential execution of the statements.
+
+<img src="{{ 'images/Sequential_Statement_Execution.png' | relative_url }}" alt="CockroachDB Parallel Statement Execution Error Mismatch" style="border:1px solid #eee;max-width:100%" />
+
+Traditional SQL engines execute the statements sequentially without affecting performance. But in case of globally-distributed CockroachDB clusters, sequential execution of statements adds to aggregate latency. This is because to maintain strong consistency across a globally-distributed cluster, CockroachDB replicates writes using the Raft [consensus protocol](https://www.cockroachlabs.com/blog/consensus-made-thrive/). The protocol requires majority of replicas to agree before a write is committed. Replicas always live on separate nodes, and communication between these nodes takes time. Executing SQL statements sequentially leads to higher cumulative communication latency. 
```

Comment:
> Traditional SQL engines execute the statements sequentially without affecting performance. But in case of globally-distributed CockroachDB clusters, sequential execution of statements adds to aggregate latency

This doesn't really make sense. Traditional SQL engines still take time to execute statements! It's just that they're usually a bit faster than us because of the reason you provide in the next few sentences. This means that they would also benefit from a feature like this, the speedup just wouldn't be as pronounced.

> Executing SQL statements sequentially leads to higher cumulative communication latency.

Executing SQL statements sequentially, especially when each statement requires long distance communication leads to higher cumulative latency.

Reply:
I think we can just say "Traditional SQL engines execute statements sequentially. But in case of globally-distributed CockroachDB cluster....". So basically, remove the "without affecting performance" part. Would that work?